environment.yml

```diff
@@ -0,0 +1,34 @@
+# To use:
+#   $ conda env create -f environment.yml  # `mamba` works too for this command
+#   $ conda activate numpy-dev
+name: numpy-dev
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - cython
+  - compilers
+  - openblas
```

Comment:
hmm. Shouldn't this be using our version of OpenBLAS?

Reply:
No, it should use the conda-forge provided one.